### ISSUE TITLE
Time select and datetime select not honoring config.time_zone value

### ISSUE THREAD

**Issue by arjunswaj** [rails/rails#19229]
Form helpers time_select and datetime_select are not honoring the timezone set in config.time_zone in application.rb. It should convert the value provided in the form to UTC and then save it. It's currently saving the value in UTC as it is.
Version: Ruby 2.2, Rails 4.2

**Comment by Qianfinland** [comment]
I have tried time_select with Ruby 2.2 and Rails 4.2, but couldn't reproduce the bug, it converts it to UTC and saves it in UTC for me.

**Comment by eileencodes** [comment]
@arjunswaj can you build a simple application (just the req models/controller action, config setting, and form) that reproduces the issue? Thanks!

**Comment by teknull** [comment]
I've tried to duplicate using Rails 4.2 and Ruby 2.2 and cannot reproduce.

**Comment by arjunswaj** [comment]
@eileencodes I've created a sample application [here](https://github.com/arjunswaj/time-select-test) 
I've set the config.time_zone value to 'Mumbai' as specified in the [API](http://apidock.com/rails/ActionView/Helpers/DateHelper/time_select). When I save the value, it takes as it is in UTC and saves it. 
For example: when I enter 07:00 in the time_select and have set the config.time_zone to Mumbai, it should be converted to 01:30 UTC and get persisted It is however persisting it as 07:00 UTC which is incorrect. The conversion happens properly for updated_at fields.
